src/Shape/Stack.elm

```diff
@@ -147,7 +147,7 @@ offsetSilhouette series =
                         |> List.transpose
                         |> List.map (List.sum << List.map Tuple.second)
             in
-            List.map2 (\( x, y ) newY -> ( -newY / 2, y + (-newY / 2) )) first ys
+            List.map2 (\( _, y ) newY -> ( -newY / 2, y + (-newY / 2) )) first ys
```

Comment:
That would be a great solution :).
I'll see if I can refactor some things so clarity is preserved yet without unused variables.

Reply:
My proposed refactoring:

````elm
offsetSilhouette : List (List ( Float, Float )) -> List (List ( Float, Float ))
offsetSilhouette allSeries =
    case allSeries of
        [] ->
            []

        firstSeries :: tailSeries ->
            let
                ySums =
                    allSeries
                        |> List.transpose
                        |> List.map (List.sum << List.map Tuple.second)

                offsetFirstSeries =
                    List.map2 
                        (\( _, y ) ySum -> ( -ySum / 2, y + (-ySum / 2) ))
                        firstSeries
                        ySums
            in
                offsetNone <| offsetFirstSeries :: tailSeries
````

Even better would be to have a Point type alias, so it can become:
````elm
offsetSilhouette : List (List Point) -> List (List Point)
````
Or two Type aliases:
````elm
offsetSilhouette : List Series -> List Series
````

Let me know what you think.

Reply:
TBH I think it's fine as it is (meaning in the diff above). I am simply still a bit split about the value of linters like elm-analyse, where sometimes fine nuance is lost in order to make code slightly more uniform. Of course many of the changes are beneficial (like removing dead branches). Some are meh (like adding a type annotation for constants - nobody really needed it). Some, on the other hand feel a bit too prescriptive.

Thinking about it a bit more, I think my ideal workflow for this would be more that it would run for a PR and would post these as a sort of automated code review - so more suggestions rather than gatekeeper. 

But that seems like a big project that's not really worth doing for this codebase. So (this rant aside) let's keep it as you've changed it so far and perhaps we can revisit some of these things in the future.